pom.xml

```
@@ -1,33 +1,33 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>3.2</version>
-        <relativePath />
+        <version>4.46</version>
```

this should be newer:
```suggestion
        <version>4.74</version>
```

---

the idea for this PR is to have it working with not too recent Jenkins (my Company is way behind) and upgrading it imply a lot of changes
But it is already used in a separate branch with a lot of other upgrade for a more recent Jenkins version (noted in PR description)